bitnami/cassandra/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: cassandra
+version: 1.0.0
```

Comment:
I think we have initialized other charts with `0.0.1`

Reply:
+1